torchvision/datasets/caltech.py

```diff
@@ -143,13 +142,11 @@ class Caltech256(VisionDataset):
             downloaded again.
     """
 
-    def __init__(self, root,
-                 transform=None, target_transform=None,
+    def __init__(self, root, transform=None, target_transform=None, transforms=None,
```

Comment:
nit: you still have `transforms` here, let's keep the behavior the same for now, without adding extra `transforms`

Reply:
Good eye. Seems I wasn't careful enough. Fixed in e1d776c.